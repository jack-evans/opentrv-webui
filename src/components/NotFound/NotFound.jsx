import React from 'react'

const NotFound = () => {
  return (
    <div className='NotFound'>
      <img style={{paddingBottom: '100px', paddingTop: '100px'}} src={require('../../images/opentrv.jpg')} alt='OpenTRV logo' />
      <h1>404 - Not Found</h1>
    </div>
  )
}

export default NotFound
