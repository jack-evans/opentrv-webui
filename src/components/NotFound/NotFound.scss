
.NotFound {
  background-color: $ui-01;
  height: 100vh;
}