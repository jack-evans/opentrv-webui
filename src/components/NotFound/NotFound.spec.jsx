import React from 'react'
import NotFound from './NotFound'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() })

describe('NotFound.jsx', () => {
  it('renders without crashing', () => {
    expect(shallow(<NotFound />)).toMatchSnapshot()
  })
})
