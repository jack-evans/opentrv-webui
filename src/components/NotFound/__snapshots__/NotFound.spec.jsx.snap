// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound.jsx renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt="OpenTRV logo"
          src="opentrv.jpg"
          style={
            Object {
              "paddingBottom": "100px",
              "paddingTop": "100px",
            }
          }
        />,
        <h1>
          404 - Not Found
        </h1>,
      ],
      "className": "NotFound",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "OpenTRV logo",
          "src": "opentrv.jpg",
          "style": Object {
            "paddingBottom": "100px",
            "paddingTop": "100px",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404 - Not Found",
        },
        "ref": null,
        "rendered": "404 - Not Found",
        "type": "h1",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="OpenTRV logo"
            src="opentrv.jpg"
            style={
              Object {
                "paddingBottom": "100px",
                "paddingTop": "100px",
              }
            }
          />,
          <h1>
            404 - Not Found
          </h1>,
        ],
        "className": "NotFound",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "OpenTRV logo",
            "src": "opentrv.jpg",
            "style": Object {
              "paddingBottom": "100px",
              "paddingTop": "100px",
            },
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404 - Not Found",
          },
          "ref": null,
          "rendered": "404 - Not Found",
          "type": "h1",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
